tests/unit/amo/reducers/test_addonsByAuthors.js

```diff
@@ -0,0 +1,128 @@
+import { ADDON_TYPE_THEME } from 'core/constants';
+import reducer, {
+  NB_OTHERS_ADDONS_BY_AUTHORS,
+  fetchOtherAddonsByAuthors,
+  initialState,
+  loadOtherAddonsByAuthors,
+} from 'amo/reducers/addonsByAuthors';
+import { fakeAddon } from 'tests/unit/amo/helpers';
+
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes properly', () => {
+      const state = reducer(undefined);
+      expect(state).toEqual(initialState);
+    });
+
+    it('ignore unrelated actions', () => {
+      const state = {
+        ...initialState,
+        // We slightly modify the state to be sure that an unrelated action
+        // does not change it.
+        byAddon: {
```

Comment:
I'm sort of confused by this name `byAddon`. What is it supposed to mean?

Reply:
I borrowed this in another reducer, review maybe. The idea is to normalize the state as described here: http://redux.js.org/docs/recipes/reducers/NormalizingStateShape.html.

Here I can ask the state: "give me other addons for the authors of this addon (slug)".